src/cmd/services/m3coordinator/downsample/samples_appender.go

```diff
@@ -137,6 +137,10 @@ func (a *multiSamplesAppender) addSamplesAppender(v samplesAppender) {
 	a.appenders = append(a.appenders, v)
 }
 
+func (a *multiSamplesAppender) addSamplesAppenders(v []samplesAppender) {
```

Comment:
Can we remove this method so callers don't need to allocate a slice? Just call it multiple times?